phpstan-baseline.neon

```diff
@@ -0,0 +1,21 @@
+parameters:
+	ignoreErrors:
+		-
+			message: "#^Unsafe usage of new static\\(\\)\\.$#"
+			count: 3
+			path: src/Client.php
+
+		-
+			message: "#^Unsafe usage of new static\\(\\)\\.$#"
+			count: 1
+			path: src/Connection/Parameters.php
+
+		-
+			message: "#^Static method Predis\\\\Command\\\\RawCommand\\:\\:create\\(\\) invoked with 0 parameters, at least 1 required\\.$#"
```

Comment:
This is on purpose to test the exception:

https://github.com/predis/predis/blob/113145f3065b8f34da2e09359f403e4db213b136/tests/Predis/Command/RawCommandTest.php#L66-L71